sfge/src/test/java/com/salesforce/graph/symbols/apex/ApexStringValueTest.java

```diff
@@ -34,19 +34,83 @@ public void testClassGetName() {
                         + "       System.debug(MyClass.class.getName());\n"
                         + "    }\n"
                         + "}";
-
         TestRunner.Result<SystemDebugAccumulator> result = TestRunner.walkPath(g, sourceCode);
-        SystemDebugAccumulator visitor = result.getVisitor();
 
-        ApexStringValue apexStringValue = visitor.getSingletonResult();
+        ApexStringValue apexStringValue = result.getVisitor().getSingletonResult();
         MatcherAssert.assertThat(apexStringValue.isIndeterminant(), equalTo(false));
     }
 
-    /** Methods that take a string and return a boolean */
+    /**
+     * ---- Test methods that no parameters and return a boolean
+     * ----------------------------------------
+     */
+    public static Stream<Arguments> booleanReturnNoParameter() {
+        return Stream.of(
+                Arguments.of(ApexStringValue.METHOD_CONTAINS_WHITESPACE),
+                Arguments.of(ApexStringValue.METHOD_IS_ALL_LOWER_CASE),
+                Arguments.of(ApexStringValue.METHOD_IS_ALL_UPPER_CASE),
+                Arguments.of(ApexStringValue.METHOD_IS_ALPHA),
+                Arguments.of(ApexStringValue.METHOD_IS_ALPHA_SPACE),
+                Arguments.of(ApexStringValue.METHOD_IS_ALPHANUMERIC),
+                Arguments.of(ApexStringValue.METHOD_IS_ALPHANUMERIC_SPACE),
+                Arguments.of(ApexStringValue.METHOD_IS_ASCII_PRINTABLE),
+                Arguments.of(ApexStringValue.METHOD_IS_BLANK),
+                Arguments.of(ApexStringValue.METHOD_IS_EMPTY),
+                Arguments.of(ApexStringValue.METHOD_IS_NOT_BLANK),
+                Arguments.of(ApexStringValue.METHOD_IS_NOT_EMPTY),
+                Arguments.of(ApexStringValue.METHOD_IS_NUMERIC),
+                Arguments.of(ApexStringValue.METHOD_IS_NUMERIC_SPACE),
+                Arguments.of(ApexStringValue.METHOD_IS_WHITESPACE));
+    }
+
+    @MethodSource(value = "booleanReturnNoParameter")
+    @ParameterizedTest(name = "{displayName}: {0}")
+    public void testIndeterminantString_BooleanReturn(String methodName) {
+        String sourceCode =
+                "public class MyClass {\n"
+                        + "    public static void doSomething(String s) {\n"
+                        + "       System.debug(s."
+                        + methodName
+                        + "());\n"
+                        + "    }\n"
```

Comment:
Ah... spotless again made this look aweful. Why doesn't it like?
+ "       System.debug(s." + methodName + "());\n"